docs/SetupAzureDatabase-1.md

```diff
@@ -0,0 +1,58 @@
+# Set up a database
+
+This guide explains how to set up a database for Scalar DL in Azure and how to configure helm charts for Scalar DL deployment.
+
+## Cosmos DB
+
+In this section, you will set up a Cosmos DB for Scalar DL.
+
+### Requirements
+
+* You must create a Cosmos DB account with `Core(SQL)` API.
+* You must change Cosmos DB `Default consistency` to `Strong`.
+* You must use Capacity mode as `Provisioned throughput`.
+
+### Steps
+
+* Create an Azure Cosmos DB account on the basis of [Azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/create-cosmosdb-resources-portal#create-an-azure-cosmos-db-account) with the above requirements.
+
+### Configure Scalar DL 
```

Comment:
BTW, Isn't it weird to talk about the custom values in Helm before going to the helm part?
I feel it should be moved to the next section. What do you think? (not saying to do it, so think)

Reply:
> I feel it should be moved to the next section

Change it to Install scalardl section in main document ? but in that case I am not sure how to separate it for different databases.

Reply:
Anyway, the current doc is weird since we start editing the custom values before really talking about the Helm charts.
I think it should be fixed somehow.
Please think about it and reorganize the docs property.

Reply:
I have separated configure Scalar DL section to a [new document](https://github.com/scalar-labs/scalar-kubernetes/blob/manual-guide-azure/docs/ConfigureScalarDL.md#configure-scalar-dl) from database document and referred it in [Helm part](https://github.com/scalar-labs/scalar-kubernetes/blob/manual-guide-azure/docs/ManualDeploymentGuideScalarDLOnAzure-1.md#steps-3). 

I will add suda-san's [recommendation](https://github.com/scalar-labs/scalar-kubernetes/pull/133#discussion_r654733923) in [Configure Scalar DL document](https://github.com/scalar-labs/scalar-kubernetes/blob/manual-guide-azure/docs/ConfigureScalarDL.md#configure-scalar-dl) after confirming the steps.